app/models/manageiq/providers/configuration_manager.rb

```diff
@@ -7,6 +7,7 @@ class ManageIQ::Providers::ConfigurationManager < ::ExtManagementSystem
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
   has_many :inventory_groups,             :dependent => :destroy, :foreign_key => "ems_id", :inverse_of => :manager
   has_many :configuration_script_sources, :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :authentications,              :dependent => :destroy, :foreign_key => "manager_id"
```

Comment:
A `ConfigurationManager` is an `ExtManagementSystem`, which includes the `AuthenticationMixin`, which already defines this `has_many :authentications` relationship.

This seems wrong.